app/_posts/2024-04-17-kuma-2-7-0.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Kuma 2.7 release with GUI improvements, shadow policies and more...
+description: Kuma 2.7 release with GUI improvements, shadow policies and more...
+date: 2024-04-17
```

Comment:
Not sure how important/necessary this is but if you need to change the date don't forget to do that!